### PR TITLE
testy_homepage

### PR DESCRIPTION
**Pull request by kamamenszu**
Hej, zrobiłam spec dla testowej strony Mercury_tours, która była podana w kilku zadaniach z selenium tutorials.

### CODE REVIEW

File: home.spec.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	beforeEach(function () {
+		homePage.get();
+	});
+
+	it('hould have home page with title Welcome: Mercury Tours', function () {
+		expect(homePage.getTitle()).toEqual("Welcome: Mercury Tours");
+	});
+
+	it('should have footer with © 2005, Mercury Interactive (v. 011003-1.01-058)', function () {
+		expect(homePage.getText()).toEqual("© 2005, Mercury Interactive (v. 011003-1.01-058)");
```
Comment (jacekokrojek) [inline home.spec.js]: I think that getText is not enough meaningful name for the function. Please change it.

File: home.spec.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	});
+
+	it('should have visable Mercury Tours logo after entering site' , function () {
+		expect(homePage.visableLogo()).toBe(true);
```
Comment (jacekokrojek) [inline home.spec.js]: By convention if funtion return boolean value it should be named isXXX.

File: home.spec.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+	});
+
+	it('should have menu items, top and left', function () {
+		expect(homePage.getLeftMenuAtIdx(2).getText()).toBe("Hotels");
```
Comment (jacekokrojek) [inline home.spec.js]: Can you change this test so it verifies all menu items in both menus. Please create a function that returns all menu elements for each menu.

File: pages/home.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+  this.getTopMenuAtIdx = function(idx) {
+    return this.topMenu.get(idx);
+  }
+
+  this.getSpecialOffersAtIdx = function(idx) {
+    return this.specialOffers.get(idx);
+  };
+  
+  /*this.clickArrow = function() {
```
Comment (jacekokrojek) [inline pages/home.js]: Please remove commented text

**Review by jacekokrojek — CHANGES_REQUESTED** [review]

**Comment by kamamenszu** [comment]
I've changed cases You asked me.

**Comment by kamamenszu** [comment]
I've changed spec for menu, using map.function.

**Review by jacekokrojek — APPROVED** [review]

**Pull request merged**